.github/workflows/tests.yml

```diff
@@ -2,6 +2,8 @@ name: Tests
 
 on:
   push:
+    branches:
```

Comment:
Are these changes really useful?

Reply:
This change makes the test not run again when doing tag